.github/workflows/pythonpackage.yml

```diff
@@ -1,6 +1,6 @@
 name: Python package
 
-on: [push]
+on: [pull_request]
```

Comment:
don't you want to run that on every push event? ie. is this going to run on every commit? Doesn't seem like it will looking at https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-event-pull_request

Reply:
let's see what happens next time. I don't really want it to run on every commit everywhere (at least I don't want to do that as default broadly if we start copying things :) )